pkg/dogstatsd/server_worker.go

```diff
@@ -1,19 +1,33 @@
 package dogstatsd
 
+import "github.com/DataDog/datadog-agent/pkg/metrics"
+
+var (
+	// defaultSampleSize is the default allocation size used to store
+	// samples when multiple a message contains multiple values. This will
```

Comment:
`samples when a message contains multiple values` instead of `samples when multiple a message contains multiple values`